examples/fluent-bit/multi-config-support/README.md

```diff
@@ -0,0 +1,83 @@
+## FireLens Example: Multiple Config support - using the Fluent Bit image with init tag
+
+This example shows you how to set multiple config files for Fluent Bit on ECS. 
+
+This example simulates a situation: A ECS Task include two containers, the app container using httpd image, and the log router container using Fluent Bit image, which will forward the logs from the App container to CloudWatch. 
+
+Based on above situation, we want to use multiple config feature to parse dummy data logs first, and forward the parsed dummy data logs to S3.
+
+
+
+### Step 1: Create config files locally
+
+**dummy-input.conf**
+
+```
+[INPUT]
+    Name 					dummy
+    Tag 	  			dummy.data
+    Dummy   			{"data":"100 0.5 true This is Demo"}
+```
+
+**dummy-filter.conf**
+
+ ```
+[FILTER]
+    Name					 parser
+    Match					 dummy.*
+    Key_Name 			 data
```

Comment:
I'd prefer if the example not use dummy input. You could use something from here: https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/ecs-log-collection

Or take a look at our internal tickets and use some redacted config inspired from those.

Reply:
Got it, will find a suitable example and rewrite it